### PR TITLE
avatar-badge done

### PR DESCRIPTION
**Pull request by kushalsourav**
avatar and badges completed

### CODE REVIEW

File: packages/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    line-height: 0.875rem;
+    height: 1rem;
+    width: 1rem;
+    top: 2%;
+    right: -10%;
+    text-align: center;
+}
+
+.status-online {
+    background-color: green;
```
Comment (ashishkatiyar2404) [inline packages/components/badge.css]: use variables for colors.
I have seen you  already use but also use here.

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

**Comment by ashishkatiyar2404** [comment]
All looks good to me.
but Speed up, deadline is near.

**Review by dipen2000 — APPROVED** [review]
Looks good man , you are consistent with the naming and you are using variables for colors and all those things. 
image

There is some problem here with the alignment do check that and other than that you are good to go.

**Comment by dipen2000** [comment]
The deadline is very near and you have a lot of ground to cover so buckle up now and complete this thing.
best of luck.

**Comment by kushalsourav** [comment]
> Looks good man , you are consistent with the naming and you are using variables for colors and all those things. image
> 
> There is some problem here with the alignment do check that and other than that you are good to go.

position absolute is adding padding automatically for it , when i remove positioning it layers properly but positioning is imp for using that component

**Pull request merged**